include/seastar/net/api.hh

```diff
@@ -450,8 +465,8 @@ public:
         return false;
     }
 
-    /** 
-     * Returns available network interfaces. This represents a 
+    /**
+     * Returns available network interfaces. This represents a
```

Comment:
I will accept this change because you didn't do many of them, but in general, please avoid "improving" the style of code which isn't related to your patch together with your actual patch.

Reply:
Fair enough: I don't make manual "improvements" to the style in unrelated changes, but his trimming is done automatically by my editor for edited files. I can turn it off for this project if you've grown fond of your trailing whitespace :).